package haptix.comm.msgs;

/// \ingroup haptix_comm_msgs
/// \interface hxModel
/// \brief Information about a model.

import "hxJoint.proto";
import "hxLink.proto";
import "hxTransform.proto";

message hxModel
{
  required string name           = 1;
  required hxTransform transform = 2;
  repeated hxLink links          = 3;
  repeated hxJoint joints        = 4;
  required bool gravity_mode     = 5;
}
